Automattic/vip-go-mu-plugins
# Adding checks for Jetpack Autoconnection staged rollout

## Description

We want to roll out Jetpack Autoconnection through Connection Pilot slowly. For now, we only attempt to connect a site to Jetpack if it's a new site.

- If it's a new site, we are picking an arbitrary GOOP Site ID and assuming everything after it is _new_.
- If it's a multisite, we can use our `is_fresh_site` logic.

We are safe to remove the changes introduced in this PR after the ramp up is complete.

## Changelog Description

### Plugin Updated: Jetpack

Preparing Jetpack Autoconnection ramp-up.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Enable Connection Pilot on a fresh site and see that it autoconnects.
3. Enable Connection Pilot on an old site and see that it doesn't autoconnect.
